packages/flutter/test/material/theme_test.dart

```diff
@@ -254,13 +255,13 @@ void main() {
     );
     await tester.pump(const Duration(milliseconds: 100)); // Halfway through the theme transition
 
-    glyphText = tester.renderObject(find.byType(RichText));
+    glyphText = tester.renderObject(find.byType(RichText, skipOffstage: false));
```

Comment:
The Icon and Chip examples IMHO are compelling examples of why what you find with a finder in a test is distinct than what gets reported to the accessibility tools. I don't think there's value in marrying them.